lib/lab_manager/app/endpoints/computes.rb

```diff
@@ -23,7 +23,9 @@ class Compute < Base
         get '/:id' do
           begin
             ids = params[:id].to_s.split(',').each(&:to_i)
-            ::Compute.find(ids.count == 1 ? params[:id] : ids).to_json
+            computes = ::Compute.find(ids.count == 1 ? params[:id] : ids)
+            Array.wrap(computes).map(&:reload_provider_data) unless params[:cached]
+            computes.to_json
```

Comment:
integration tests for `cache` params are missing. Btw, current usuage `cached: 'false'` used in test doesn't work.